up-linux-streamer-plugin/README.md

```diff
@@ -0,0 +1,173 @@
+# up-linux-streamer-plugin
+
+## The Concept
+
+```mermaid
+classDiagram
+
+namespace EclipseRepos {
+    class LinuxStreamerZenohPlugin {
+        libzenoh_plugin_linux_streamer.so
+    }
+    class ZenohRouter
+    class up-transport-zenoh-rust
+    class up-transport-vsomeip-rust
+    class up-rust
+}
+
+
+namespace ExternalRepos {
+    class vsomeip {
+        vsomeip.so
+    }
+    class zenoh
+    class ZenohRouter {
+        zenohd
+    }
+}
+
+note for LinuxStreamerZenohPlugin "Deployed to vehicle"
+note for vsomeip "Deployed to vehicle"
+note for ZenohRouter "Deployed to vehicle"
+
+LinuxStreamerZenohPlugin o-- up-transport-zenoh-rust: uses
+LinuxStreamerZenohPlugin o-- up-transport-vsomeip-rust: uses
+
+up-transport-zenoh-rust ..> up-rust: builds upon
+up-transport-vsomeip-rust ..> up-rust: builds upon
+
+up-transport-vsomeip-rust ..> vsomeip: builds upon
+up-transport-zenoh-rust ..> zenoh: builds upon
+LinuxStreamerZenohPlugin ..> ZenohRouter: loaded by
+```
+
+## Using the plugin
+
+### Build the plugin
+
+```bash
+LD_LIBRARY_PATH=<your/path/to/vsomeip/lib> VSOMEIP_LIB_DIR=<your/path/to/vsomeip/lib> cargo build
+```
+
+### Copy plugin and configs to standalone folder
+
+```bash
+mkdir -p my/new/standalone/zenohd/path
+cp target/debug/libzenoh_plugin_up_linux_streamer.so my/new/standalone/zenohd/path/
+cp up-linux-streamer-plugin/DEFAULT_CONFIG.json5 my/new/standalone/zenohd/path/
+cp -r up-linux-streamer-plugin/vsomeip-configs my/new/standalone/zenohd/path/
+```
+
+### Using the plugin
+
+Because up-transport-zenoh-rust uses minimum supported Rust version (**MSRV**) of 1.74.0 and up-rust uses MSRV of 1.72.1, we need to build Zenoh from source in order to get a compatible zenohd (Zenoh Router). The following steps describe how to do so.
```

Comment:
Is it still an issue? In fact, I'm thinking of building your own zenohd in the project instead of from Zenoh source code.

Reply:
This is a good idea. Opened #43 for it